sdk/resourcemanager/Azure.ResourceManager.Core/src/Azure.ResourceManager.Core.csproj

```diff
@@ -8,7 +8,7 @@
       This is a beta preview vesion.  This version uses a next-generation code generator that introduces important breaking changes, but also new features (such as intuitive authentication, custom HTTP pipeline, distributed tracing and much more).
     </Description>
     <PackageTags>azure;management;resource</PackageTags>
-    <NoWarn>$(NoWarn);AZC0008;AZC0001;AZC0107;CA2214;CA1036;CA1067;CA1065;SA1028</NoWarn>
+    <NoWarn>$(NoWarn);AZC0008;AZC0001;AZC0107;CA2214;CA1036;CA1067;CA1065;SA1028;CA1310</NoWarn>
```

Comment:
lets not add more of these since it just adds work on us later, lets fix whatever CA1310 is.